R/analysis.R

```diff
@@ -12,9 +13,34 @@ Analysis <- function(Functions, condition, replications, fixed_objects, prepare
     #  and number of replications desired
 
     # Call prepare function once per condition if provided
+    prepare_error_seed <- NULL
+    prepare_Random.seed <- NULL
     if(!is.null(prepare)) {
+
+        # Restore seed if debugging prepare
+        if(!is.null(load_seed_prepare))
+            .GlobalEnv$.Random.seed <- load_seed_prepare
+
+        # Ensure .Random.seed exists (initialize RNG if needed)
+        else if(!exists(".Random.seed", envir = .GlobalEnv))
```

Comment:
This should be moved to .on.Attach() as it affects the other .Random.seed instances too